src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1389,11 +1389,22 @@
   
     <Field Name="InitializerExpressionOpt" Type="BoundExpression" Null="allow"/>
   </Node>
-  
+
   <Node Name="BoundDelegateCreationExpression" Base="BoundExpression">
```

Comment:
There was useless whitespace removed here, but there is only one line. Perhaps CodeFlow does not behave well for lines where there was only whitespace and the whitespace was all that was removed.